be/src/exec/pipeline/olap_table_sink_operator.cpp

```diff
@@ -81,6 +81,37 @@ bool OlapTableSinkOperator::pending_finish() const {
     return false;
 }
 
+bool OlapTableSinkOperator::is_epoch_finishing() const {
+    return pending_finish();
+}
+
+Status OlapTableSinkOperator::set_epoch_finishing(RuntimeState* state) {
+    _is_epoch_finished = true;
+    if (_is_open_done) {
```

Comment:
why not just `_is_opened`...

Reply:
it's the reuse `set_finishing` method. I don't change this any more.